langserver/jedi.py

```diff
@@ -140,3 +136,29 @@ def load_source(path) -> str:
             )
 
         return jedi.api.Script(*args, **kwargs)
+
+def get_module_search_paths(module_name, script_file_path):
+    '''
+    Provides an ordered list of directories in the workspace to search for the 
+    given 'module_name', starting from the directory that the script is operating on. 
+    
+    This mimics Jedi's modifications of sys.path that it uses during module resolution.
+    See: 
+    https://sourcegraph.com/github.com/sourcegraph/jedi/-/blob/jedi/evaluate/imports.py#L237:9
+    '''
+    for parent in traverse_parents(script_file_path):
+        if os.path.basename(parent) == module_name:
```

Comment:
This is correct for 99.99% of the cases, but I recommend adding a note that there are some edge cases for pull PEP-0235 compatibility with respect to case (https://www.python.org/dev/peps/pep-0235/). Definitely not worth fixing now, though!